frontend/src/metabase/collections/components/PinnedItemCard/PinnedItemCard.tsx

```diff
@@ -60,6 +60,7 @@ const TOOLTIP_MAX_WIDTH = 450;
 
 const DEFAULT_DESCRIPTION: Record<string, string> = {
   card: t`A question`,
+  metric: t`A metric`,
```

Comment:
I am not sure about this change because it seems that the `PinnedItemCard` is never used for metrics.